main.go

```diff
@@ -80,6 +128,32 @@ func New(config ...Config) fiber.Handler {
 		if cfg.XFrameOptions != "" {
 			c.Set(fiber.HeaderXFrameOptions, cfg.XFrameOptions)
 		}
+		if cfg.CrossOriginEmbedderPolicy != "" {
+			c.Set("Cross-Origin-Embedder-Policy", cfg.CrossOriginEmbedderPolicy)
```

Comment:
Should we add new consts for these headers if they are not exists?

Reply:
I'm trying to make it match what `helmet.js` does. #42 suggested stricter values, but while doing so I noticed we are missing some headers that `helmet.js` supports.

Reply:
I think we can add them to fiber core 🤔

Reply:
@efectn Yeah, I feel like this middleware should be part of the core. It's quite simple implementation wise.